api-testing/functional/tests/cypress/integration/01-institution/patch-institutions-institution.ts

```diff
@@ -0,0 +1,32 @@
+describe("Institution api", () => {
+  it("PATCH - Update institution details", () => {
+    cy.fixture("token").then((data) => {
+      cy.request({
+        method: Cypress.env("patch"),
+        url: Cypress.env("update_institution_url"),
+        auth: { bearer: data.access_token },
+        body: {
+          primaryContactEmail: "ryan@gov.bc.ca",
```

Comment:
Are these values planned to be moved to the test fixtures?

Reply:
Added into fixtures.